<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentDialog</class>
 <widget class="QDialog" name="AddStudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Додати Студента</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>321</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Додати Студента</string>
   </property>
   <widget class="QTextEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="adressEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ім'я:</string>
    </property>
   </widget>
   <widget class="QLabel" name="facultyLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Факультет:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="facultyEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="groupLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Група:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="groupEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="adressLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Адреса:</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="submitButton">
      <property name="text">
       <string>Додати</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="revertButton">
      <property name="text">
       <string>Очистити</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Закрити</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
